### ISSUE TITLE
Extend to textbox's

### ISSUE THREAD

**Issue by CloudPires** [MaxArt2501/share-this#28]
Is it possible to extend this to text boxes?

**Comment by MaxArt2501** [comment]
Short answer: no.

Long answer: the library relies on some [document-wide events](https://github.com/MaxArt2501/share-this/blob/master/src/core.js#L7) to check whether a selection is present. And the selection itself is retrieved using the [Selection API](https://developer.mozilla.org/en-US/docs/Web/API/Selection_API).

The problem here is that, although the `selectionchange` event *is* fired when you select something in a textarea... it doesn't happen in Firefox and I'd say that, in fact, it should *not* happen. So it's not reliable.
But what's worse is that when something is selected inside a textarea (or any other input field), `window.getSelection().getRangeAt(0)` yields either `null` or a collapsed range that has nothing to do with the textarea.

In short, as far as I know it's not possible to know the visual boundaries of a selection inside a textarea. Not without some unacceptable approximations.
